### PR TITLE
update guide.md

### PR DESCRIPTION
**Pull request by karani12**
Fixed a few typos for the guide.md file.

### CODE REVIEW

File: docs/blog2/src/guide.md [hunk 1]
```diff
@@ -77,11 +77,11 @@ The flowchart above may help you understand how the plugin works and it's design
 
 ## Collecting Articles and Generating Info
 
-To get started, the plugins will pick those pages which neeeds to be tricted as articles. This is only the first step to drop those pages you don’t want.
+To get started, the plugins will pick those pages which need to be treated as articles. This is only the first step to drop those pages you don’t want.
 
 ::: note
 
-By default, all the pages generated from Markdown files but not homepage will be included as articles.
+By default, all the pages generated from Markdown filesa apart from homepage be included as articles.
```
Comment (Mister-Hope) [inline docs/blog2/src/guide.md:84]: What's `filesa`? Is it a exist english word?

**Review by Mister-Hope — CHANGES_REQUESTED** [review]
I think this is a typo

File: docs/blog2/src/guide.md (hunk 1 above)
Comment (karani12) [reply docs/blog2/src/guide.md:84]: sorry didn't see that.Let me fix that

**Comment by Mister-Hope** [comment]
Thanks for it, if you have time, welcome to improve other docs as well.🙂

**Pull request merged**